src/app/quiz/_components/step/quiz-step2.tsx

```
@@ -100,14 +100,15 @@ export const QuizStep2 = ({ onNext }: QuizStepProps) => {
         quizContent={
           <div className='flex justify-center'>
             <div
-              className='relative inline-block cursor-pointer'
+              className='relative w-full max-w-[26.875rem] min-w-[22.5rem] cursor-pointer'
               onClick={handleImageClick}
             >
               <Image
                 src='/images/quiz/quiz2-bg.png'
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)